docs/en/observability/quickstarts/collect-data-with-aws-firehose.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[collect-data-with-aws-firehose]]
+= Quickstart: Collect data with AWS Firehose
+
+preview::[]
+
+In this quickstart guide, you'll learn how to send logs and metrics to Elastic using AWS Firehose.
+
+The AWS Firehose receiver is based on a CloudFormation template that can collect all available CloudWatch logs and metrics for your AWS account.
+
+This approach requires minimal configuration as the CloudFormation template creates a Firehose stream, enables CloudWatch metrics collection, and sets up account level subscription filter for CloudWatch log groups to send logs to Elastic via Firehose.
+
+NOTE: Optionally, you can download the CloudFormation template to review and modify the default settings. For more information, check the 
```

Comment:
Here I'd also clarify that downloaded template could be used in customer's existing IaC (infrastructure as code) setup.